### DISCUSSION TITLE
Add support in Renovate for `asdf-plugin-manager`

### DISCUSSION THREAD

**Discussion by MPV** [asdf-community/asdf-plugin-manager#25] [Ideas]
Hi,
This project looks very promising! 🙏 

Would you also be interested in Renovate having support for `asdf-plugin-manager`?

I've made a discussion there too:
- https://github.com/renovatebot/renovate/discussions/24667

I'm not sure if I know how (or have the time at the moment) to build the support in Renovate, but I figured I'd raise it here to get the ball rolling. 🎈 

For comparison, this issue outlines some examples of how to add an ASDF plugin in general to Renovate:
- https://github.com/renovatebot/renovate/discussions/24582

**Comment by MPV** [comment]
@aabouzaid what are your thoughts on this? 🙏🏻

**Reply by aabouzaid** [reply]
That's a pretty good idea :rocket: 
I'm not so familier with JS (incase we want add it as renovate datasource), but I will try to take a look soon.

**Comment by MPV** [comment]
Not the same, but I made this PR there for at least bumping a `.tool-versions` file with a newer `asdf-plugin-manager` version:
- https://github.com/renovatebot/renovate/pull/25539

**Comment by MPV** [comment]
I have a PoC for this now:
- https://github.com/MPV/dotfiles/blob/d4a7ac900d6204a742bd0dd67b960e488ef898fe/renovate.json#L7-L18

Giving results like this:
- https://github.com/MPV/dotfiles/pull/121

Happy to hear/discuss any feedback or ideas you might have on this.
